app/assets/javascripts/admin/addon/controllers/admin-customize-themes-show.js

```diff
@@ -476,4 +488,26 @@ export default class AdminCustomizeThemesShowController extends Controller {
   editColorScheme() {
     this.router.transitionTo("adminCustomize.colors.show", this.colorSchemeId);
   }
+
+  @action
+  onLightColorChange(name, value) {
+    const color = this.model.colorPalette.colors.find((c) => c.name === name);
+    color.hex = value;
+  }
+
+  @action
+  onDarkColorChange(name, value) {
```

Comment:
I'm wondering if we can just pass the color object to this callback instead of having to loop through again in the controller to retrieve the object.

Reply:
done!